package.json

```diff
@@ -1,28 +1,165 @@
+    "test:watch": "jest --coverage --watchAll",
+    "typescript": "tsc -p tsconfig.json --noEmit"
   },
   "keywords": [
     "react-native",
+    "ios",
+    "android",
     "speech",
     "voice",
     "asr",
-    "sad",
-    "vad",
-    "android"
+    "tts"
+  ],
+  "repository": "https://github.com/spokestack/react-native-spokestack",
+  "contributors": [
+    "Noel Weichbrodt <noel@spokestack.io>",
+    "Timmy Willison <4timmywil@gmail.com>"
   ],
-  "repository": {
-    "type": "git",
-    "url": "git://github.com/spokestack/react-native-spokestack.git"
-  },
   "license": "Apache-2.0",
+  "bugs": {
+    "url": "https://github.com/spokestack/react-native-spokestack/issues"
+  },
+  "homepage": "https://github.com/spokestack/react-native-spokestack#readme",
   "peerDependencies": {
+    "react": "*",
     "react-native": ">=0.60.0"
+  },
+  "dependencies": {},
+  "devDependencies": {
+    "@commitlint/config-conventional": "^11.0.0",
+    "@react-native-community/bob": "^0.16.2",
+    "@react-native-community/eslint-config": "^2.0.0",
+    "@release-it/conventional-changelog": "^2.0.0",
+    "@types/jest": "^26.0.0",
+    "@types/react": "^17.0.0",
+    "@types/react-native": "^0.63.37",
+    "commitlint": "^11.0.0",
+    "concurrently": "^5.3.0",
+    "eslint": "^7.2.0",
+    "eslint-config-prettier": "^6.11.0",
+    "eslint-plugin-prettier": "^3.1.3",
+    "husky": "^4.2.5",
+    "jest": "^26.0.1",
+    "pod-install": "^0.1.0",
+    "prettier": "^2.0.5",
+    "react": "^17.0.1",
+    "react-native": "^0.63.4",
+    "release-it": "^14.2.2",
+    "typedoc": "^0.19.2",
+    "typedoc-plugin-markdown": "^3.0.11",
+    "typescript": "^4.1.2"
+  },
+  "jest": {
+    "preset": "react-native",
+    "modulePathIgnorePatterns": [
+      "<rootDir>/example/node_modules",
+      "<rootDir>/lib/"
+    ],
+    "coveragePathIgnorePatterns": [
+      "/node_modules/",
+      "/example/",
+      "/tasks/",
+      "/test/"
+    ]
+  },
+  "husky": {
+    "hooks": {
+      "commit-msg": "commitlint -E HUSKY_GIT_PARAMS",
+      "pre-commit": "concurrently --raw \"npm run lint\" \"npm run typescript\""
+    }
+  },
+  "commitlint": {
+    "extends": [
+      "@commitlint/config-conventional"
+    ]
+  },
+  "eslintConfig": {
+    "extends": [
+      "@react-native-community",
+      "prettier"
+    ],
+    "rules": {
+      "eqeqeq": "error",
+      "no-fallthrough": "off",
+      "no-shadow": "off",
+      "prettier/prettier": "error",
+      "react-native/no-inline-styles": 0
+    }
+  },
+  "eslintIgnore": [
+    "node_modules/",
+    "coverage/",
+    "lib/",
+    "docs/"
+  ],
+  "prettier": {
+    "arrowParens": "always",
+    "semi": false,
+    "singleQuote": true,
+    "tabWidth": 2,
+    "trailingComma": "none",
+    "useTabs": false
+  },
+  "@react-native-community/bob": {
+    "source": "src",
+    "output": "lib",
+    "targets": [
+      "commonjs",
+      "module",
+      "typescript"
+    ]
+  },
+  "release-it": {
```

Comment:
Honestly, I don't it's necessary to publish to cocoapods. React Native projects install RN libs from the node modules folder.

Reply:
the `RNSpokestack`, or i guess now it's `react-native-spokestack` is a podspec that requires explicit releasing

Reply:
What makes you says that? As I said, I'm pretty sure it's not necessary since RN projects pull from local files. I checked a few of the react native libs we use and they are not on cocoapods.

Reply:
a podspec is definitionally a pod that is released by the creator and available as a cocoapods package. https://cocoapods.org/pods/RNSpokestack

Reply:
I understand you have successfully published it to cocoapods, but there are other uses for a podspec (including installing a pod from node modules and declaring pod dependencies like spokestack-ios). That does not mean it should be published to cocoapods. In fact, what's published as a pod on cocoapods right now is essentially useless as it is not a standalone package, which is probably why you had so much trouble linting it when publishing the first time.

Reply:
This is why RN project Podfiles have this line...

```ruby
require_relative '../node_modules/react-native/scripts/react_native_pods'
```